docker-api/src/main/java/com/yahoo/vespa/hosted/dockerapi/metrics/Gauge.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.dockerapi.metrics;
+
+/**
+ * @author freva
+ */
+public class Gauge implements MetricValue {
```

Comment:
Would have been great if Gauge, Counter, and more(?) could have been put in a more appropriate module than docker-api, which is supposed to contain only docker-java internals. Unfortunately I don't immediately see what module that would be, or whether a new module would be required.

Reply:
True, but because this is for host-admin only and the dependency between host-admin modules, the only way to solve this is to either create a new module or to have it here :neutral_face: